### DISCUSSION TITLE
Expose the `GetActiveScriptOrModule` method.

### DISCUSSION THREAD

**Discussion by scgm0** [sebastienros/jint#1807] [Q&A]
I have a method that passes in js from C#, it needs to get the module path when js calls this method, although it is possible to parse the module path from a stack string, however I found out that there is already a `GetActiveScriptOrModule` method, is it possible to expose it directly?

**Comment by lahma** [comment]
`GetActiveScriptOrModule` is very internal implementation details to engine so I wouldn't want to expose that. Why would you need to get the module path?

**Reply by scgm0** [reply]
Roughly, I have a method that accepts bbcode strings that allows [img]{path}[/img] tags, and I'd like to implement support for relative paths for it, rather than only absolute paths.
